app/views/reports/_auditing_report.html.haml

```diff
@@ -46,19 +46,25 @@
               =label_tag "title", t(:documents)[:title]+t(:actions)[:required], class: "col-sm-3 control-label"
               .col-sm-9= text_field_tag "title", title, class: "form-control"
             .form-group
-              = label_tag "start_date", t(:reports)[:start_date]+t(:actions)[:required], class: "col-sm-3 control-label"
-              .col-sm-9
-                = text_field_tag "start_date", nil, class: "datepicker form-control", readonly: true
+              = label_tag "start_date", t(:reports)[:start_date]+t(:actions)[:required], class: "col-sm-4 col-form-label"
+              .input-group.datetimepicker.date.col-sm-8{ id: 'start_date', data: {target_input: 'nearest' }}
```

Comment:
The IDs on these datepickers are the same as the inputs which breaks the unique element IDs standard

Reply:
No, the ids are not being duplicated. One is the id and the other is just setting the target.